src/inverse-kinematics/include/iDynTree/InverseKinematics.h

```diff
@@ -144,12 +147,15 @@ class iDynTree::InverseKinematics
     bool setFloatingBaseOnFrameNamed(const std::string &floatingBaseFrameName);
 
     /*!
-     * Sets the robot configuration
+     * Sets the robot current configuration
      *
      *
      * @param baseConfiguration  transformation identifying the base pose with respect to the world frame
      * @param robotConfiguration the robot configuration
      *
+     * @note the size (and order) of jointConfiguration must match the joints in the model, not
+     * in the consideredJoints variable
+     *
      * @return true if successful, false otherwise.
      */
     bool setRobotConfiguration(const iDynTree::Transform& baseConfiguration,
```

Comment:
Not directly related to this PR, but this method is a bit confusing, we should consider renaming it (`setCurrentRobotConfiguration` ? ).